test/factory/common_factory_tests.dart

```diff
@@ -21,7 +21,7 @@ void commonFactoryTests(ReactComponentFactoryProxy factory) {
 
   group('passes children to the component when specified as', () {
     dynamic getJsChildren(ReactElement instance) =>
-        getProperty(instance.props, 'children');
+        getProperty(instance.props as dynamic, 'children');
```

Comment:
Why is this needed?

Reply:
dart1

Reply:
@kealjones-wk since this will be going into the `5.0.0-wip` branch _(which will be Dart 2 only)_, this can be removed.

Reply:
Yeah this was before we removed the travis config for dart1 :P